<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>460</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Download SVG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Thickness:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Shape:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Rectangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TriPrism</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Units:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>81</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Inches</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Centimetres</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>540</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Length:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>580</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>620</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="colorText">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>500</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lengthText">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>530</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="heightText">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>580</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="widthText">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>620</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>911</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
   <widget class="QLabel" name="finger_slot_label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>560</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Finger Slot: </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fing_slot_check">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>550</y>
      <width>105</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fing_width_label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>600</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Finger Width: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fing_width_line">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>590</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="thickness_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>.25</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="depth_text">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>660</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="depth_lbl">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>670</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Depth:</string>
    </property>
   </widget>
   <widget class="QLabel" name="side_lbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Side:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="side_combo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>93</width>
      <height>25</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>All</string>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Side 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Side 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Side 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Side 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Side 5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Side 6</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
